src/AppBundle/Entity/ConnectionRepository.php

```diff
@@ -153,4 +153,24 @@ public function isUserConnectionExists(User $user1, User $user2)
 
         return $qb->getQuery()->getSingleScalarResult()? true: false;
     }
-}
+
+    /**
+     * @param $days
+     * @param $numSent
+     * @return array
+     */
+    public function findCreatedDaysAgo($days, $numSent)
+    {
+        return $this
+            ->createQueryBuilder('c')
+            ->where('DATE_DIFF(:now, c.createdAt) >= :days')
```

Comment:
@jongotlin no this is not a good idea. I do not want all connections ever made to get this email. @rifeman2007 what is you thought on this?